webview/src/plots/components/ribbon/RibbonBlock.tsx

```diff
@@ -23,7 +23,20 @@ export const RibbonBlock: React.FC<RibbonBlockProps> = ({
 }) => {
   const exp = revision.group?.replace(/[[\]]/g, '') || revision.revision
 
-  return (
+  const tooltipContent = (
+    <table className={styles.columnsTable}>
+      <tbody>
+        {revision.firstThreeColumns.map(({ path, value }, ind) => (
+          <tr key={ind}>
```

Comment:
Might be better to use the path as the key as the order can change, but the path won't. It's not that expensive to let the row re-render, but it's still good practice to avoid using the order as keys.

Reply:
Done!